docs/data_statistics.md

```diff
@@ -19,19 +19,20 @@ Long-term statistics consist of two tables. One contains the metadata, the other
 
 ### Statistics Meta
 
-This table contains the metadata describing the source. Statistics are currently limited to be derived from entities, and so `statistic_id` is currently equivalent to `entity_id`. When an entity ID is changed, the statistic ID is automatically updated.
+This table contains the metadata describing the source. Statistics is either derived from entities, in which case `statistic_id` is equivalent to the `entity_id`, or imported from an external source. Statistics imported from an external source is similar to `entity_id`, but uses a `:` instead of a `.` as a delimiter between the domain and object ID.
```

Comment:
```suggestion
This table contains the metadata describing the source. Statistics are either derived from entities, in which case `statistic_id` is equivalent to the `entity_id`, or imported from an external source. Statistics imported from an external source are similar to `entity_id`, but use a `:` instead of a `.` as a delimiter between the domain and object ID.
```